client/web/src/routes/home/home.template.ts

```diff
@@ -2,5 +2,178 @@ import { html } from '@microsoft/fast-element';
 import type { Home } from './home';
 
 export const HomeTemplate = html<Home>`
-  <!-- insert template code here -->
+<div class="wrapper">
```

Comment:
You will be able to preview and configure component in the configurator app at any point - no need to add preview content to the app itself.

Reply:
Ok, looks like these are meant for testing - I'd describe them in the how to test section above instead of including them in the PR